Fixes ubuntu desktop image error of package not found

harshitgarg22 opened the pull request:
## Summary

Fixes #473 .

## Implementation Notes :hammer_and_pick:

* Adds universe repository in ubuntu
* Adds ca-certificates package to pull from opensuse repo

## External Dependencies :four_leaf_clover:

* N/A

## Breaking API Changes :warning:

* N/A

*Simply specify none (N/A) if not applicable.*

harshitgarg22 commented:
@dhuang-esl can you please confirm if this works?

ustiugov commented on scripts/setup_system.sh:
```diff
@@ -24,6 +24,10 @@
 
 # Add skopeo sources
  . /etc/os-release
+sudo apt-get -y install curl >> /dev/null
```
Please combine lines 27 and 29

ustiugov reviewed (changes requested):
Please remove the merge commit, and rebase your branch on top of main. Please read the docs/contributing*.md for the details how to do that.

harshitgarg22 commented:
> Please remove the merge commit, and rebase your branch on top of main. Please read the docs/contributing*.md for the details how to do that.

Please check if this is fine. Thanks for the heads up.

ustiugov commented:
GitHub still shows 2 commits in this PR

ustiugov commented on scripts/setup_system.sh:
```diff
@@ -24,6 +24,9 @@
 
 # Add skopeo sources
```
This comment is for the curl command, please move it down

ustiugov reviewed (changes requested):
- [ ] Need to have 1 commit, please use 'git rebase -i' to get rid of the merge commit. 
- [ ] Move the comment as specified

harshitgarg22 commented:
> * [ ]  Need to have 1 commit, please use 'git rebase -i' to get rid of the merge commit.
> 
>     * [ ]  Move the comment as specified

The two commits were for the original change and the combining of lines. I've squashed all changes into one now. Moved the comment above the curl command.
